scripts/src/main/resources/scripts/functions

```diff
@@ -780,21 +780,28 @@ function doReplaceExtractedFile() {
   local filename="${1/*\//}"
   local target_dir="${DEVON_IDE_HOME}"
   local target="${DEVON_IDE_HOME}/${filename}"
+  local migration="${DEVON_IDE_HOME}/scripts/migration"
+  local curr_version
+  curr_version="$("${target_dir}/scripts/devon" -v)"
```

Comment:
Should we remove the option to specify the current version of the IDE with `devon -v` as first parameter at all? 

Further, should we have the newly created `.devon.software.version` automatically updated after each release?